tex/latex/fks/fksmeta.sty

```diff
@@ -80,9 +80,35 @@
 \met@probheader & %
 \Roman{solvedbatch} & \textnormal{\%} & $\Sigma$ \\}
 
+% For switching between FYKOS and Vyfuk functionality across texmf files.
```

Comment:
Souhlasím, také by se mi to více líbilo.

Ano, používají. Resp. nevím jak jinak napsat:
```
if(fykos) {...}
if(fykos && czech) {...}
if(fykos && english) {...}
if(vyfuk) {...}
```